File: apps/venture/lib/venture/slide/fork.ex
defmodule Venture.Slide.Fork do

  @derive {Jason.Encoder, except: [:next, :notes]}
  defstruct type: "fork", location: %{story: "", index: 0}, next: nil, content: "", notes: "",
            paths: [], background: nil, class: nil, attribution: nil, align: ""

end

File: apps/venture_web/assets/js/records/Fork.js
import Immutable from 'immutable';

export default class Fork extends Immutable.Record({
  type: 'fork',
  location: {story: undefined, index: undefined},
  next: undefined,
  paths: [],
  background: undefined,
  class: undefined,
  content: undefined,
  notes: '',
  attribution: undefined,
  align: ''
}) { }
